pkg/controllers/edgeconnect/secret/secret.go

```diff
@@ -46,6 +46,11 @@ func PrepareConfigFile(ctx context.Context, instance *edgeconnectv1alpha1.EdgeCo
 		cfg.RootCertificatePaths = append(cfg.RootCertificatePaths, consts.EdgeConnectMountPath+"/"+consts.EdgeConnectCustomCertificateName)
 	}
 
+	if instance.IsK8SAutomationEnabled() {
+		cfg.RootCertificatePaths = append(cfg.RootCertificatePaths, consts.EdgeConnectServiceAccountCAPath)
```

Comment:
The certificate should always be added. Only the secret shall be only added if automation is enabled.

Reply:
fixed in 70372af6